### PR TITLE
Ebnt 299 integrity error

### PR DESCRIPTION
**Pull request by ElmiRakh**

### CODE REVIEW

File: src/ebonite/core/objects/core.py [hunk 1]
```diff
@@ -808,6 +818,9 @@ def __getattr__(self, item: str):
     def save(self):
         self._meta.save_model(self)
 
+    def has_children(self):
```
Comment (mike0sv) [inline src/ebonite/core/objects/core.py:821]: ?

Comment (mike0sv) [reply src/ebonite/core/objects/core.py:821]: it's not even EboniteObject

File: tests/client/client_common.py [hunk 2]
```diff
@@ -264,6 +264,7 @@ def test_delete_image__not_only_meta_ok(ebnt: Ebonite, image_to_delete: Image, m
 def test_build_and_run_instance(ebnt: Ebonite, regression_and_data, container_name, mock_env):
     reg, data = regression_and_data
     mock_env = ebnt.meta_repo.create_environment(mock_env)
+    mock_env = mock_env.bind_as(mock_env)
```
Comment (mike0sv) [inline tests/client/client_common.py:267]: this does nothing

File: src/ebonite/core/objects/core.py (hunk 1 above)
Comment (ElmiRakh) [reply src/ebonite/core/objects/core.py:821]: Model is not an EboniteObject?

File: tests/client/client_common.py (hunk 2 above)
Comment (ElmiRakh) [reply tests/client/client_common.py:267]: Leftovers. Removed

**Pull request merged**